Core/src/org/sleuthkit/autopsy/casemodule/NoCurrentCaseException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2018 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.casemodule;
+
+import org.openide.util.NbBundle;
+
+/**
+ *
+ * Exception thrown when no current case is available
+ */
+@NbBundle.Messages({
+        "NoCurrentCaseException.message=No open case is available."
+    })
+public class NoCurrentCaseException extends Exception {
+
+    NoCurrentCaseException() {
```

Comment:
Instead of providing a no arguments constructor, please provide one constructor with a message parameter, and one constructor with a message parameter and a cause parameter. You will find plenty of examples in other Autopsy code.